content/swagger/resources/attributes/routes/attributes_code_options.yaml

```diff
@@ -125,3 +125,47 @@ post:
       $ref: "#/responses/415Error"
     422:
       $ref: "#/responses/422Error"
+patch:
+  summary: Update/create several attribute options
+  operationId: "patch_attributes__attribute_code__options"
+  tags:
+    - Attribute options
+  x-versions:
+    - 2.1
+  description: This endpoint allows you to update several attribute options at once.
+  x-body-by-line: Contains several lines, each line is an attribute option in JSON standard format
+  parameters:
+    - $ref: '#/parameters/attribute_code'
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/AttributeOption"
+        x-examples:
+          x-example-1:
+            '{"code":"black", "labels":{"en_US": "Black","fr_FR": "Noir"}}'
```

Comment:
add `"attribute": "attribute_code"`in one line please, to show that it is optional
and a second line with not the same `attribute_code` to show that it creates an error